databuilder/tables/beta/tpp.py

```diff
@@ -195,5 +195,5 @@ class ons_cis(EventFrame):
 
     visit_date = Series(datetime.date)
     visit_num = Series(int)
-    nhs_data_share = Series(int)
+    nhs_data_share = Series(str)
```

Comment:
This should have a categorical constraint if we're going to model it as a categorical. Something like:
```py
nhs_data_share = Series(
    str,
    constraints=[
        Constraint.Categorical(["opt-out", "opt-in"]),
    ],
)
```

But I wonder if this wouldn't be better modelled as a boolean. Something like `is_opted_in_to_nhs_data_share`?

Reply:
Yes, a boolean probably makes sense.  I wonder if that would make it confusing for the ONS team who are using a mix of cohort-extractor and databuilder?
I guess we can ask them

Reply:
Well that's neat! (I think you meant "0=opt-in" above, right?)

Reply:
yes :facepalm:

Reply:
I will check the db to make sure there really are only 0, 1 or null in that column first though

Reply:
Confirmed :point_up: